pkg/pillar/cmd/zedmanager/handlezedrouter.go

```diff
@@ -37,6 +37,15 @@ func MaybeAddAppNetworkConfig(ctx *zedmanagerContext,
 				m.GetStatsIPAddr.String(), aiConfig.CollectStatsIPAddr.String())
 			changed = true
 		}
+		if m.CloudInitUserData != aiConfig.CloudInitUserData {
```

Comment:
Note that this cleartext user data is going away real soon and we'll only use the encrypted user-data.

Reply:
Good point. For now I have no ideas about encryption here (probably you can point me onto docs about it?). Let me find more information about it and implement it on Eden side for testing.
But is it broken for now or it is some kind of plan for removing? If it is in plan, what particular fields will be removed here?

Reply:
Your addition of the CipherData comparison is sufficient.